pkgs/applications/science/biology/trimmomatic/default.nix

```diff
@@ -0,0 +1,57 @@
+{ ant
+, fetchFromGitHub
+, jdk11_headless
+, jdk8_headless
+, jre_minimal
```

Comment:
Why three JDKs?

Reply:
As early as jdk8 was needed in nativeBuildInputs for the code to compile. Later jdk-s did not work. I used jre_minimal to wrap the resulting jar. I experimented with different jdk options to reduce the closure size. jdk8 errored out because of missing jlink binary, so jdk11 was the smallest.

Maybe that's an overkill, and I should just use vanilla jre_minimal without the jdk11 thingy?

Reply:
Usually you should stick with the most recent version of the inputs you are selecting. Smaller versions are reserved for softwares that are not yet updated.

If you can run it on the JDK19, use JDK19.

Reply:
If you need to force JDK8, then be consistent and force it everywhere.

Reply:
OK, I can try to find a way to use the same java to compile and to wrap.